params.go

```diff
@@ -131,6 +131,8 @@ type RequestParams struct {
 	ProxyBuffersNumber   int    `json:"proxybuffersnumber,omitempty" yaml:"proxybuffersnumber,omitempty"`
 	ClientBodyBufferSize string `json:"clientbodybuffersize,omitempty" yaml:"clientbodybuffersize,omitempty"`
 	LoadBalanceAlgorithm string `json:"loadbalance,omitempty" yaml:"loadbalance,omitempty"`
+	AuthSecret           string `json:"authsecret,omitempty" yaml:"authsecret,omitempty"`
+	VerifyDepth          int    `json:"verifydepth,omitempty" yaml:"verifydepth,omitempty"`
```

Comment:
What is this verify depth?

Reply:
It is verification depth in client certificate chain.